invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/globals/site.overrides

```diff
@@ -422,6 +422,29 @@ dl.details-list {
   text-align: center;
 }
 
+.text-align-right {
```

Comment:
does it make sense to add this rule to the common `invenio-theme`, maybe?

Reply:
if not already there ? I am not sure, but I think I saw a class handling this